app/views/layouts/application.html.erb

```diff
@@ -39,8 +39,11 @@
             <%= image_tag "signup_avatar.jpeg", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
             <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
         <% if user_signed_in? %>
-              <%= link_to "My bookings", bookings_path, class:"dropdown-item" %>
-              <%= link_to "Log out",  destroy_user_session_path, class: "dropdown-item", data: {turbo_method: :delete} %>
```

Comment:
Here there will be conflicts with the pr I just merged. 
The "my bookings, and "add new location" are now in the nav bar as links and no longer in the drop down menu. 
So we would need to add the my locations link as a list to the nav-items.